### PR TITLE
Updated packaging scripts

### PR DESCRIPTION
**Pull request by alanhartless**
Updated the packaging scripts to be "exclusive" rather than "inclusive" to catch newly added files/directories to root.

**Testing**
Build the packages and makes sure they install, work, and/or update.

### CODE REVIEW

File: build/package_release.php [hunk 1]
```diff
@@ -111,8 +110,10 @@
 // Post-processing - ZIP it up
 chdir(__DIR__ . '/packaging');
 
+system("rm -f ../packages/{$version}.zip ../packages/{$version}-update.zip");
+
 echo "Packaging Mautic Full Installation\n";
-system('zip -r ../packages/' . $version . '.zip addons/ app/ bin/ media/ themes/ translations/ vendor/ favicon.ico .htaccess index.php LICENSE.txt robots.txt > /dev/null');
+system('zip -r ../packages/' . $version . '.zip . -x@../excludefiles.txt');
```
Comment (mbabker) [inline build/package_release.php]: I'd suggest putting the ` > /dev/null` back at the end here unless you want the dump in your console of all the packaged files.

File: build/package_devhead.php [hunk 2]
```diff
@@ -47,4 +36,6 @@
 
 // Step 5 - ZIP it up
 echo "Packaging Mautic\n";
-system('zip -r ../packages/mautic-head.zip addons/ app/ bin/ media/ themes/ translations/ vendor/ favicon.ico .htaccess index.php LICENSE.txt robots.txt > /dev/null');
+chdir(__DIR__ . '/packaging');
+
+system('zip -r ../packages/mautic-head.zip . -x@../excludefiles.txt');
```
Comment (mbabker) [inline build/package_devhead.php]: Same here

**Comment by alanhartless** [comment]
Thanks! Yea I removed them when testing and forgot to put them back.  Doing so now.

**Comment by escopecz** [comment]
Tested with Chad's test bundle. I inserted the bundle to the /addon directory, clicked Install/Upgrade Addons button and nothing happened. Cache has to be cleared first. Is that OK?

The bundle works and updates on file save.

**Pull request merged**